docs/SX_Merge_PR_Review_Strat.md

```diff
@@ -1,7 +1,7 @@
 S->X PR review key points:
 
 * "40s" Literals
-    * Check for "40s" or "40X" in file names or content.
+    * Check for "40s" or "40S" in file names or content.
```

Comment:
This is supporsed to be 40X, right?

Reply:
I think the green text is correct.
It warns that "40S" in a file on x-dv is suspicious.

Reply:
Oh, yes. My mistake.